sources/simulators.swift

```diff
@@ -29,14 +59,21 @@ func getBootedSimulators() throws -> [String] {
     }
 
     let devices = json["devices"] as? [String: [[String: String]]] ?? [:]
-    let bootedIDs = devices
-        .flatMap { $1 }
-        .filter { $0["state"] == "Booted" }
-        .flatMap { $0["udid"] }
 
-    if bootedIDs.isEmpty {
-        throw SimulatorFetchError.noBootedSimulators
+    var simulators = [Simulator]()
+    for topName in devices.keys {
+        if let deviceTypes = devices[topName] {
```

Comment:
Can this be `for (topName, deviceTypes) in devices`?

Reply:
yeah much cleaner. thanks =)